include/unix/osd.h

```diff
@@ -37,6 +37,8 @@
 #include <unistd.h>
 #include <errno.h>
 
+#define FI_WAIT_MUTEX_CRITSEC FI_WAIT_MUTEX_COND
```

Comment:
Why does this still exist?  Seems like it should have been removed as part of the conversions to `#ifdef`s.  Using it makes the code a bit shorter but much less clear.  It also tramples the `FI_` namespace in a confusing way.